Makefile

```diff
@@ -70,7 +70,7 @@ test_ci:
 	stestr run --concurrency $(CONCURRENCY)
 
 test_randomized:
-	python3 -m unittest discover -s test/randomized -v
+	python3 -m unittest discover -s test/randomized -t . -v
```

Comment:
Should this be on L58?

Reply:
oops yep

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/4942/commits/27c7bfa479e477f1238e8681dd3cc5107b74091f